docs/modules/ROOT/pages/explanations/decisions/subscription-tracking.adoc

```diff
@@ -0,0 +1,71 @@
+= Subscription tracking
+
+== Problem
+
+We need to track the yearly subscriptions bought for OpenShift 4 clusters.
+Subscription count is required for a monthly report to RedHat.
+RedHat subscriptions are based on (v)CPU count.
+We need to cover the difference of the yearly subscription count and the actual CPU count with monthly subscriptions.
+
+Buying yearly subscriptions saves us money compared to buying monthly subscriptions.
+
+It also adds a small amount of risk if a customer decides to cancel their clusters before the year is over.
+
```

Comment:
```suggestion
```

I'd remove these lines, they don't add value to this problem statement (business things)